moveit_core/planning_request_adapter/src/planning_request_adapter.cpp

```diff
@@ -140,7 +144,10 @@ bool PlanningRequestAdapterChain::adaptAndPlan(const planning_interface::Planner
       fn = [&adapter = *adapters_[i], fn, &added_path_index = added_path_index_each[i]](
                const planning_scene::PlanningSceneConstPtr& scene, const planning_interface::MotionPlanRequest& req,
                planning_interface::MotionPlanResponse& res) {
-        return callAdapter(adapter, fn, scene, req, res, added_path_index);
+        res = callAdapter(adapter, fn, scene, req, added_path_index);
+        // If any adapter fails, return false. Otherwise, the pipeline would continue onward anyway and the failure
```

Comment:
No need to document the old behavior now that it's fixed. Simply "Abort pipeline and return error code in case of failure" is sufficient, imo.